### ISSUE TITLE
[request]

### ISSUE THREAD

**Issue by miketweaver** [IronicBadger/docker-booksonic#1]
It would be a good idea to redirect users from the default tomcat page to the booksonic page.

IE: Users visit: `http://192.168.1.56:8769/` and it redirects to: `http://192.168.1.56:8769/booksonic/index.view`

The redirect will need to useable on any port and hostname.

**Comment by tuxflo** [comment]
Yes or at least a message in the README.md would be sufficient. The /booksonic also has to be set in the app.

**Comment by ironicbadger** [comment]
Noted. Can you guys submit a pull request to fix as I'm away at the moment?

**Comment by miketweaver** [comment]
Submitted. Pull request #2
